addons/stock_account/models/account_move.py

```diff
@@ -132,35 +132,11 @@ def _stock_account_prepare_anglo_saxon_out_lines_vals(self):
                 amount_currency = sign * line.quantity * price_unit
 
                 # Add interim account line.
-                lines_vals_list.append({
-                    'name': line.name[:64],
-                    'move_id': move.id,
-                    'partner_id': move.commercial_partner_id.id,
-                    'product_id': line.product_id.id,
-                    'product_uom_id': line.product_uom_id.id,
-                    'quantity': line.quantity,
-                    'price_unit': price_unit,
-                    'amount_currency': -amount_currency,
-                    'account_id': debit_interim_account.id,
-                    'display_type': 'cogs',
```

Comment:
One could easily change the values thanks to the `display_type`, either with a computed field (easy, readable, probably better than anything you are trying to do), or in `create`

Reply:
Hello @william-andre 

This case is not of adding or changing the `display_type` of the line.

Actually we are trying to fix how the real-time valuation cogs journal items are created and can we analyze / fix / improve / leverage-on the creation of cogs lines.

Could take a new look the new proposal to this PR.

Best Regards.

Reply:
Hello @william-andre do you have any news regarding this PR.

Best Regards.